util.js

```diff
@@ -943,6 +943,14 @@ export const handleDropJsonItem = async (item) => {
   }
   return null;
 };
+export const handleBlobUpload = async (name, blob, progress) => {
+  const formData = new FormData();
+  formData.append(name, blob, name);
+  const hashes = await doUpload(`https://ipfs.webaverse.com/`, formData, progress);
```

Comment:
This should be a constant.

Reply:
const ipfsFileURL = 'https://ipfs.webaverse.com/';
const ipfsFolderURL = 'https://ipfs.webaverse.com/ipfs';